iati/settings/base.py

```diff
@@ -536,7 +536,7 @@
 
 # Base URL to use when referring to full URLs within the Wagtail admin backend -
 # e.g. in notification emails. Don't include '/admin' or a trailing slash
-BASE_URL = 'http://iatistandard.org'
+WAGTAILADMIN_BASE_URL = 'http://iatistandard.org'
```

Comment:
Are we sure BASE_URL is no longer used anywhere? I would rather we retain as well (we don't lose anything having a setting that's not used, do we?) instead of potentially breaking something else that depends on it. But of course, if you are confident nothing will break, all good.

Reply:
Also, kindly remove Alex L and Nik's emails from Admin emails. Potentially replace with mine and yours.

Reply:
Yes am sure, it is being renamed to WAGTAILADMIN_BASE_URL, and i searched the code, BASE_URL was not being referred to anywhere else within the code

Reply:
It's used here by wagtail 
`

Gets the base URL for the wagtail admin site. This is set in `settings.WAGTAILADMIN_BASE_URL`,
--
12 | which was previously `settings.BASE_URL`.
13 | """
14 |  
15 | admin_base_url = getattr(settings, "WAGTAILADMIN_BASE_URL", None)
16 | if admin_base_url is None and hasattr(settings, "BASE_URL"):

`